### PR TITLE
Accessibility edits

### PR DESCRIPTION
**Pull request by imcbride**
Here's a roll-up of recent accessibility tweaks. For #80

### CODE REVIEW

**Comment by imcbride** [comment]
I've added a few fixes for the datepicker and slideshows.

File: src/js/events-datepicker.js [hunk 1]
```diff
@@ -64,3 +64,37 @@ if (datePicker) {
...
+const pikaTitle = document.querySelector('.pika-title');
+
+if (pikaTitle) {
+  pikaTitle.setAttribute('aria-atomic', 'true');
+}
+
+const pikaTable = document.querySelector('.pika-table');
+
+if (pikaTable) {
+  pikaTable.removeAttribute('cellpadding');
```
Comment (zebapy) [inline src/js/events-datepicker.js]: Curious why these attributes were removed? Granted they shouldn't be used on our tables since CSS should handle it.

Comment (imcbride) [reply src/js/events-datepicker.js]: Correct, that's the reason. https://my2.siteimprove.com/Accessibility/541441/Pages/Issue?CheckId=58&CriterionChapter=1.3.1

File: src/js/events-datepicker.js [hunk 2]
```diff
@@ -64,3 +64,37 @@ if (datePicker) {
     }
   });
 }
+
+const selects = ['month', 'year'];
+
+for (let i in selects) {
+  if (selects.hasOwnProperty(i)) {
+    let select = 'pika-select-' + selects[i];
```
Comment (zebapy) [inline src/js/events-datepicker.js]: I'd use `const` for vars that won't be reassigned. the `let` in `for` is fine though.

File: src/js/events-datepicker.js [hunk 3]
```diff
@@ -64,3 +64,37 @@ if (datePicker) {
     }
   });
 }
+
+const selects = ['month', 'year'];
+
+for (let i in selects) {
+  if (selects.hasOwnProperty(i)) {
```
Comment (zebapy) [inline src/js/events-datepicker.js]: Why is this check necessary?

Comment (imcbride) [reply src/js/events-datepicker.js]: The linter complains "The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype". The if statement prevents the case where `i` is assigned an index beyond the scope of the array. This _should_ never happen, but potentially could if the array is modified.

File: src/js/events-datepicker.js (hunk 2 above)
Comment (imcbride) [reply src/js/events-datepicker.js]: Fixed.

**Pull request merged**